components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/display/DisplayToolbar.kt

```diff
@@ -573,6 +573,10 @@ internal class DisplayToolbar(
         }
     }
 
+    fun onStop() {
+        menuView.dismissMenu()
```

Comment:
@mauryat to test this you could make  `MenuButton#menu` internal and visible for testing :)  It's OK to change visibility as long as we don't expose properties to consumers!
 
`MenuButton.kt`
```koltin
@VisibleForTesting
    internal var menu: BrowserMenu? = null
```

On the `DisplayToolbarTest.kt`  you could do something like:

```kotlin
        val menuView = extractMenuView(displayToolbar)
        
        menuView.menu = mock()
        displayToolbar.onStop()
        
        verify(menuView.menu).dismiss()
```

Or check some of the tests on `DisplayToolbarTest.kt` to get some inspiration :)